packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -61,6 +61,12 @@ const Item = ({ label, children, className }) => (
   </span>
 )
 
+const canSupportPrivateNetwork = (pool, pif) =>
+  (pif.isBondMaster || pif.physical || pif.vlan !== -1) &&
+  pif.mode !== 'None' &&
+  !pif.isBondSlave &&
+  pif.$host === (pool && pool.master)
```

Comment:
Can you explain all these tests? Maybe add a comment?

Reply:
On it. :)